backend/schema/sumtype.go

```diff
@@ -0,0 +1,70 @@
+package schema
+
+import (
+	"fmt"
+	"strings"
+
+	schemapb "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/schema"
+	"google.golang.org/protobuf/proto"
+)
+
+type SumType struct {
+	Pos Position `parser:"" protobuf:"1,optional"`
+
+	Comments []string `parser:"@Comment*" protobuf:"2"`
+	Name     string   `parser:"'sumtype' @Ident Equals" protobuf:"3"`
```

Comment:
This should be just `'='` by adding `=` to `Punct`.

Reply:
oo lala, thank you!